core/build.gradle.kts

```diff
@@ -0,0 +1,119 @@
+import com.saveourtool.processbuilder.configureDetekt
+import com.saveourtool.processbuilder.configureDiktat
+
+plugins {
+    kotlin("multiplatform")
+}
+
+kotlin {
+    jvm {
+        jvmToolchain(11)
+        withJava()
+        testRuns["test"].executionTask.configure {
+            useJUnitPlatform()
+        }
+    }
+    js(IR) {
+        browser {
+            commonWebpackConfig {
+                cssSupport {
+                    enabled.set(true)
+                }
+            }
+        }
+    }
+
+    macosArm64()
```

Comment:
Didn't remember this target in `save-cli`, does it influent on some of our platform dependent algorithms?

Reply:
There actually doesn't influent our platform-dependent algorithms, while it makes it easier to run tests on MacBook M1